src/pages/en/reference/api-reference.md

```diff
@@ -86,6 +86,8 @@ Astro files have the following interface:
 
 ```ts
 export interface AstroInstance {
+  file: string;
```

Comment:
Do we want to add some comments like we do for `MarkdownInstance`?

```suggestion
  /* The file path of this file */
  file: string;
```

Reply:
Thanks for the suggestion!